app/furniture/marketplace/vendor_representative.rb

```diff
@@ -0,0 +1,26 @@
+class Marketplace
+  class VendorRepresentative < Record
+    self.table_name = :marketplace_vendor_representatives
+
+    belongs_to :marketplace, inverse_of: :vendor_representatives
+    has_one :room, through: :marketplace
+    belongs_to :person
```

Comment:
Record creation is failing because this is not optional. Observed this while playing with the branch locally.

Reply:
Weird; I was able to create a `VendorRepresentative` through the UI...

I'll add a `spec/marketplace/system/vendor_representatives_system_spec.rb` and a bit more tests tonight before merging.

Reply:
Certainly as written now this class requires to be associated to a `person` to be valid.

Reply:
yea, adding the spec caused it to fail until I put this in... Now I'm wondering why I could do it last night...